frontend/actions/review.ts

```diff
@@ -11,7 +11,7 @@ export function useGetReviewRequestsForUser(isActive = true) {
     },
     ErrorInfo
   >(
-    `/api/v2/reviews?${qs.stringify({
+    `/api/v2/reviews/?${qs.stringify({
```

Comment:
As `active` is a query param on the `/reviews` endpoint, I wouldn't expect to see the additional `/` here.

Should it not be left as was?